This is the revised version of files/macro_tweak.patch for
xorg-server 1.8.99.904 and newer.

Upstream-Status: Pending

Signed-off-by: Yu Ke <ke.yu@intel.com>

diff --git a/xorg-server.m4 b/xorg-server.m4
index bdecf62..040fdb8 100644
--- a/xorg-server.m4
+++ b/xorg-server.m4
@@ -28,10 +28,12 @@ dnl
 # Checks for the $1 define in xorg-server.h (from the sdk).  If it
 # is defined, then add $1 to $REQUIRED_MODULES.
 
+m4_pattern_allow(PKG_CONFIG_SYSROOT_DIR)
+
 AC_DEFUN([XORG_DRIVER_CHECK_EXT],[
 	AC_REQUIRE([PKG_PROG_PKG_CONFIG])
 	SAVE_CFLAGS="$CFLAGS"
-	CFLAGS="$CFLAGS -I`$PKG_CONFIG --variable=sdkdir xorg-server`"
+	CFLAGS="$CFLAGS -I$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=sdkdir xorg-server`"
 	AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 #include "xorg-server.h"
 #if !defined $1
